about.html

```diff
@@ -0,0 +1,105 @@
+        </article>
+        <p class="text-align-center about-footer">Send an e-mail to info@concertswebsite.com for more info</p>
+        <p class="text-align-center"><strong>Call us on 00 987 65432</strong></p>
+    </section>
+    <section class="padding-top-20px background-color-white about-section-2 about-section-border">
+      <article class="text-align-center w-50-percent margin-auto">
+        <h2>The Story Behind The Logo</h2>
+        <p class="font-family-lato font-weight-normal">One hunderd of creative contestans participated in this competition, each one came up with own best design, but there had to be one winner. The hunderd logos were shortlisted to 50 at the first stage. At the second stage, there were even stricter crieteria, only 10 contestants made it to the final stage, some of world's best graphics experts voted on the winner who's logo was selected for this website!</p>
+        <div class="about-image-container margin-auto align-items-center">
+          <img class="logo-image margin-auto" src="./assets/logo.png" alt="logo image">
+        </div>
+      </article>
+    </section>
+    <section class="padding-top-20px background-color-white about-section-3 about-section-border">
+      <article class="text-align-center">
+        <h3>Our Ear<span class="underline-red">lie</span>r Concerts</h3>
+        <p>Below you can see how helarious our two erlier concerts were back in 2016 and 2012</p>
+        <div class="about-parties-container color-white align-items-center display-flex flex-direction-row justify-content-center font-weight-bolder">
+          <div class="party-2016-container">
+            <h2>2016</h2>
+            <p>France, Paris June 26th</p>
+          </div>
+          <div class="party-2012-container">
+            <h2>2012</h2>
+            <p>UK, London April 5th</p>
+          </div>
+        </div>
+      </article>
+     </section>
+    </main>
+
+    <footer class="margin-top-18px">
+      <div class="background-272a31 padding-top-20px">
+        <h1 class="text-align-center color-d3d3d3  underline-red">Contributors</h1>
+
+        <div class="display-flex flex-direction-row width-50-percent footer-top-div justify-content-space-around padding-left-right-10-percent">
+          <i class="fas fa-drum color-d3d3d3"></i>
+          <i class="fab fa-wordpress color-d3d3d3"></i>
+          <i class="fas fa-trademark color-d3d3d3"></i>
+          <i class="fas fa-ad color-d3d3d3"></i>
+          <i class="fas fa-guitar color-d3d3d3"></i>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly remove the contributors from the about page it's not supposed to be there according to the [project design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

- [ ] Please kindly do the same to the tickets page too